### ISSUE TITLE
Support for multiple database connections 

### ISSUE THREAD

**Issue by LyOuYang** [vanna-ai/vanna#285]
consider connecting multiple databases for querying relevant data？

**Comment by zainhoda** [comment]
Thanks -- this is something that we've been thinking about for a while. What are the multiple databases that you're connecting to?

Is your use case that the user should be able to ask a question and then Vanna will decide which database connection to query? 

Do you have human analysts who query multiple databases currently?

**Comment by leo10816** [comment]
Can Vanna be integrated with Trino, considering that Trino has the capability for multiple database connections?

**Comment by LyOuYang** [comment]
Yes, I would like multiple tables from different databases to be able to be joined together for querying, just like multiple tables within a single database. Since we have multiple systems corresponding to multiple databases, it is often necessary to connect to multiple databases for querying when retrieving data for analysis.

**Comment by lawsonX** [comment]
> Thanks -- this is something that we've been thinking about for a while. What are the multiple databases that you're connecting to?
> 
> Is your use case that the user should be able to ask a question and then Vanna will decide which database connection to query?
> 
> Do you have human analysts who query multiple databases currently?

Yes! In my case, I have separate databases for metadata (schema tables) and business data tables. The schema tables contain information about the names of the databases where the business data tables reside. A human engineer will look up in the schema database firstly, then he will know which database and tables contain what he wants. This is my case.

**Comment by prabhakaran-sg** [comment]
> > Thanks -- this is something that we've been thinking about for a while. What are the multiple databases that you're connecting to?
> > Is your use case that the user should be able to ask a question and then Vanna will decide which database connection to query?
> > Do you have human analysts who query multiple databases currently?
> 
> Yes! In my case, I have separate databases for metadata (schema tables) and business data tables. The schema tables contain information about the names of the databases where the business data tables reside. A human engineer will look up in the schema database firstly, then he will know which database and tables contain what he wants. This is my case.

Same in my case. Is the feature available now or any plans for that ?

**Comment by sinjup** [comment]
@zainhoda , I need to connect to multiple SQLite databases, where the user can ask a question, and Vanna should be able to determine which database to connect to and execute the query to retrieve the results.
